package.json

```diff
@@ -167,7 +167,7 @@
   },
   "dependencies": {
     "axios": "^0.24.0",
-    "codemaker-sdk": "^1.0.0"
+    "codemaker-sdk": "^1.1.0"
```

Comment:
Actually with ^, it will track the latest major version, so we don't have to update this everytime.